source/MaterialXFormat/File.cpp

```diff
@@ -141,6 +141,32 @@ bool FilePath::exists() const
 #endif
 }
 
+FilePath FilePath::getNormalized() const
+{
+    FilePath path(*this);
```

Comment:
What's the motivation for copy constructing the result path from the original, since it will be cleared below?  Perhaps we can just use a default constructor below?

Reply:
This is to copy over any other data members including type. I was thinking of explicitly copying over type, but this seems safer in case there are any changes to FilePath members in the future.

Reply:
Ah, good point, and this choice makes sense to me.